packages/jumpstarter-driver-corellium/jumpstarter_driver_corellium/corellium/api_test.py

```
@@ -0,0 +1,215 @@
+
+    requests_mock.get(f'https://api-host/api/v1/instances/{instance.id}/state', status_code=status_code, text=data)
+    api = ApiClient('api-host', 'api-token')
+    api.session = Session('session-token', '2022-03-20T01:50:10.000Z')
+
+    with pytest.raises(CorelliumApiException) as e:
+        api.read_instance_state(instance)
+
+    assert msg in str(e.value)
+
+
+def test_destroy_instance_state_ok(requests_mock):
```

Ruff (0.8.2)

Redefinition of unused `requests_mock` from line 4

(F811)